systemd/mpathcount.service

```diff
@@ -8,4 +8,5 @@ StandardInput=socket
 StandardOutput=null
 StandardError=journal
 ExecStart=/usr/bin/sh -c '. /etc/xensource-inventory; while dd of=/dev/null bs=4096 count=1 status=none conv=noerror; do /opt/xensource/sm/mpathcount.py; done'
+ExecStartPost=/bin/bash -c 'if [ ! -f "/dev/shm/mpath_status" ]; then /opt/xensource/sm/mpathcount.py; fi'
```

Comment:
I think this, if we do indeed need it, should `ExecStartPre` so that it will not interfere with the pipeline in the `ExecStart`

Reply:
I believe we need it. After RPU, if there is no path change event, NRPE or some component like NRPE won't detect the path status. Yes, I can move it to ExecStartPre. That's okay with me.

Reply:
Hi @MarkSymsCtx I have fixed this issue. could we continue to move on?